commerce.install

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * @file
+ * Contains install and update functions for Commerce.
+ */
+
+/**
+* Implements hook_requirements().
+*/
+function commerce_requirements($phase) {
+  $requirements = [];
+  if ($phase == 'install') {
+    if (!extension_loaded('bcmath')) {
+      $requirements['commerce'] = [
+      'description' => t("Commerce requires you to enable the following PHP extension: bcmath"),
```

Comment:
Wrong indentation for 'description' and 'severity'. The error message can be inside single quotes. The message should be 'Commerce requires the bcmath PHP extension.'